components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -2803,6 +2868,43 @@ private LocalAndOutboundAuthenticationConfig getLocalAndOutboundAuthenticationCo
         }
     }
 
+    /**
+     * Reads and sets AppRoleMappingConfigurations from the database.
+     *
+     * @param applicationId Application ID of the application.
+     * @param connection Database connection.
+     * @return AppRoleMappingConfig[] Array of AppRoleMappingConfigurations.
+     * @throws SQLException SQLException if an error occurs while reading from the database.
+     */
```

Comment:
check formatting 
```suggestion
     * Reads and sets AppRoleMappingConfigurations from the database.
     *
     * @param applicationId Application ID of the application.
     * @param connection    Database connection.
     * @return AppRoleMappingConfig[] Array of AppRoleMappingConfigurations.
     * @throws SQLException SQLException if an error occurs while reading from the database.
     */
```

Reply:
Addressed in 4cdf9672bdd6a4d6060995024d14bd9dc79a1168